src/lib/core/fiber.c

```diff
@@ -1561,6 +1566,12 @@ cord_add_garbage(struct cord *cord, struct fiber *f)
 		cord->garbage = f;
 }
 
+void
+cord_shutdown_is_started(void)
```

Comment:
Usually like that we call functions without side-effects which just return true/false. Here it would be rather `cord_on_shutdown_start()` or something.